app/assets/images/payment_icons/nationalebioscoopbon.svg

```diff
@@ -0,0 +1,31 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-nationalebioscoopbon">
```

Comment:
Can we remove all instances of `id`? For example, these don't seem to be needed:

`id="Logo"`
`id="Shape"`

Reply:
Done.